frontend/src/app/modules/availability/new-slot/schedule-list-item/schedule-list-item.component.ts

```diff
@@ -54,12 +56,16 @@ export class ScheduleListItemComponent extends BaseComponent implements OnInit {
         this.itemChange.emit();
     }
 
+    isTwelveHoursFormat() {
+        const user = JSON.parse(localStorage.getItem('user')!) as ILocalUser;
```

Comment:
You cannot use localStorage in components

Reply:
I created method in UserService that gets local user from localStorage. Then I used it in SlotDefinitionComponent for getting time format of current user.